Kernel/Memory/PageDirectory.cpp

```diff
@@ -61,6 +61,7 @@ ErrorOr<NonnullLockRefPtr<PageDirectory>> PageDirectory::try_create_for_userspac
             }
         }
 
+#if ARCH(X86_64)
```

Comment:
This change looks like it might've been meant for a different commit, as it doesn't match this commit's commit message.

Reply:
I kinda sneaked this one in, but will make it a different commit :)